StudyBuddy/app/src/main/java/uw/studybuddy/Events/EventCreation.java

```diff
@@ -89,6 +89,31 @@ public void onClick(View v) {
                 String course = courseCreate.getText().toString().trim();
                 String description = descriptionCreate.getText().toString().trim();
                 String location = locationCreate.getText().toString().trim();
+
+                String subject = subjectCreate.getText().toString().trim();
+/*
+        if(!TextUtils.isEmpty(course) && !TextUtils.isEmpty(description) && !TextUtils.isEmpty(location) && !TextUtils.isEmpty(subject)){
+            dataMap.put("course", course);
+            dataMap.put("description", description);
+            dataMap.put("location", location);
+            dataMap.put("subject", subject);
+
+
+            mDatabase.push().setValue(dataMap).addOnCompleteListener(new OnCompleteListener<Void>() {
```

Comment:
We wanted to keep all database-related methods in the `FirebaseInstance` class. We already have a method in `FirebaseInstance` called `addNewEventToDatabase`that does this.